conversant/personas/watch-sales-agent/config.json

```diff
@@ -35,12 +35,12 @@
                     "bot": "Not a problem at all. The Omega Seamaster is another example of a classic, luxury dive watch. Most people know it from the James Bond movies."
                 },
                 {
-                    "user": "that sounds amazing thanks!",
-                    "bot": "I'll add the Omega Seamaster to your cart. Is there anything else I can assist you with?"
+                    "user": "that sounds amazing thanks! how much is it? ",
+                    "bot": "It's $7,500. Would you like me to add the Omega Seamaster to your cart?"
                 },
                 {
-                    "user": "nah I'm good, bye!",
-                    "bot": "You're most welcome. I hope you have a wonderful day."
```

Comment:
IMO we should add this last interaction in again at the end.

Reply:
Added back!
[f543fdb](https://github.com/cohere-ai/sandbox-conversant-lib/pull/76/commits/f543fdb894061164d3d2dbfdbe1b40ea80a4c195)